msal/src/test/java/com/microsoft/identity/client/PromptTest.java

```diff
@@ -54,6 +54,13 @@ public void testOpenIdConnectParameterConsent() {
         Assert.assertEquals(promptValue, OpenIdConnectPromptParameter.CONSENT);
     }
 
+    @Test
+    public void testOpenIdConnectParameterCreate() {
+        prompt = Prompt.CREATE;
+        final OpenIdConnectPromptParameter promptValue = prompt.toOpenIdConnectPromptParameter();
+        Assert.assertEquals(promptValue, OpenIdConnectPromptParameter.CREATE);
```

Comment:
the params here probably need to be swapped (IIRC, the first one is expected and the second one is actual)